pages/reference/base-images/base-images.md

```diff
@@ -17,15 +17,15 @@ excerpt: Docker images maintained by balena
   - amd64
   - i386
 - Multiple Distributions:
-  - [Debian](https://www.debian.org/): stretch (9), buster (10), bullseye (11), bookworm (12) and sid
-  - [Alpine](https://alpinelinux.org/): 3.11, 3.12, 3.13, 3.14 and edge
-  - [Ubuntu](https://www.ubuntu.com/): xenial (16.04), bionic (18.04), focal (20.04), hirsute (21.04) and impish (21.10)
-  - [Fedora](https://getfedora.org/): 32, 33, 34 and 35
+  - [Debian](https://www.debian.org/): buster (10), bullseye (11), bookworm (12) and sid
+  - [Alpine](https://alpinelinux.org/): 3.12, 3.13, 3.14, 3.15, 3.16 and edge
+  - [Ubuntu](https://www.ubuntu.com/): xenial (16.04), bionic (18.04), focal (20.04), impish (21.10), kinetic (22.10) and jammy (22.04)
```

Comment:
Not that it makes any difference, but I guess for consistency it might be nice to list these in the "right order"? i.e. "impish, jammy and kinetic" rather than "impish, kinetic and jammy" :shrug: